components/org.wso2.carbon.identity.api.server.dcr/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/ApplicationDTO.java

```diff
@@ -81,6 +81,7 @@ public class ApplicationDTO  {
     private String requestObjectEncryptionMethod = null;
     private String softwareStatement = null;
     private  Map<String, Object> additionalAttributes;
+    private String extAllowedAudience;
```

Comment:
Does this mean external allowed audience ?

Reply:
Yes. But what we mean by that is its external property for DCR. We register an oauth app through DCR. Role audience is a property of the SP, not the oauth app. Hence we have been following this as a pattern.

Reply:
Ack. I think it's better to rename it as `externalAllowedAudience` instead of `extAllowedAudience`, as it's more clear. WYT ?

Reply:
This is not a new approach. We have been following this as a practice for all external parameters. 

FYR : https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/05e8852de6cf860948b39ece3d76ab2dd8a46090/components/org.wso2.carbon.identity.api.server.dcr/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/ApplicationDTO.java#L36